website/blog/2020-03-16-7.9.0.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title:  "7.9.0 Released: Smaller preset-env Output, Typescript 3.8 Support and a New JSX Transform"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2020-03-13 0:00:00
+categories: announcements
+share_text: "Babel 7.9.0 Released"
+---
+
+While preparing the Babel 8.0.0 [release](https://github.com/babel/babel/issues/10746) (in the next months), we just finished working on a new minor release which includes updates related to preset-modules, TS 3.8, Flow, and JSX!
+
+A few months ago, [Jason Miller](https://twitter.com/_developit) started working on [`@babel/preset-modules`](https://github.com/babel/preset-modules): an experiment to greatly reduce bundle sizes when using the [`module`/`nomodule` pattern](https://css-tricks.com/differential-serving/). We are excited to announce that its functionality has now been merged into `@babel/preset-env`! This means that its optimizations can be applied to all `preset-env` targets values, without a separate preset.
+
+*Note: These optimizations will be enabled by default in Babel 8. They can be manually enabled in Babel 7.9 by passing in the option `{bugfixes: true}` to `preset-env`.*
+
+This release also has full support for [TypeScript 3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html), which introduced explicit type-only imports and exports (i.e. `export type { foo }`), and for [Flow 0.120](https://github.com/facebook/flow/blob/master/Changelog.md#01201), which introduced the `declare` modifier for class fields (i.e. `class C { declare foo: string }`).
+
+We also worked with the React team to provide a new [JSX transform](https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md), which will make it possible for React and React-like libraries to further optimize the creation of JSX elements with the addition of the `jsx` function vs. `React.createElement`.
+
+Lastly, `@babel/parser` now supports two additional ECMAScript proposals: [Record & Tuple](https://github.com/tc39/proposal-record-tuple) and [Module Attributes](https://github.com/tc39/proposal-module-attributes). Please note that this is only parser support, and the transforms are still being worked on.
```

Comment:
May only refer to `record-tuple` only since we are holding off module-attributes.